src/main/proto/com/google/api/codegen/samplegen/v1p2/sample_config_v1p2.proto

```diff
@@ -86,6 +86,15 @@ message SampleSpecProto {
   // Definition of how to handle the RPC response, specified as an
   // ordered list of statements.
   repeated ResponseStatementProto response = 9;
+
+  // A list of the types samples to generate from this
+  // configuration. The elements can be any of "standalone" and
+  // "incode/XXX", where XXX is an arbitrary run of alphanumeric
+  // characters (these are used to lexicographically sort multiple
+  // in-code samples for the same RPC before including them in the
+  // library source code). If not specified or empty, defaults to
+  // ["standalone"]
+  repeated string sample_type = 10
```

Comment:
This is not compiling - please add a comma after `10`

Reply:
Oops! Done